osutil/mkfs/mkfs.go

```diff
@@ -106,6 +106,12 @@ func mkfsExt4(img, label, contentsRootDir string, deviceSize, sectorSize quantit
 	var cmd *exec.Cmd
 	if os.Geteuid() != 0 {
 		// run through fakeroot so that files are owned by root
+		fakerootLib := os.Getenv("FAKEROOT_LIB")
```

Comment:
Hm wonder if maybe it's more flexible to add a `FAKEROOT` env variable that holds the fakeroot invocation, eg.
```
FAKEROOT='fakeroot --lib=/foo/bar/baz' snap prepare-image ...
```
We have the code to parse this like a shell would to (see strutil/shlex package).

Reply:
I of course leave this up to you, the snapd maintainers, to decide which approach is best, but I personally feel that just passing the library path is cleaner - although less flexible indeed. Do we expect to use other fakeroot options in the nearest time?